lib/vanagon/project.rb

```diff
@@ -101,6 +101,15 @@ def get_replaces
       replaces.flatten.uniq
     end
 
+    def get_service(name)
```

Comment:
I think adding the new method makes the calls in the service files way cleaner. And i'll add comments